### PR TITLE
feat(translation): add Esperanto translation

### PR DESCRIPTION
**Pull request by kodisto**
For issue [ #439](https://github.com/gitpoint/git-point/issues/439), translation of Esperanto

### CODE REVIEW

File: src/auth/screens/language-settings.js [hunk 1]
```diff
@@ -35,6 +35,10 @@ export default [
     name: 'Русский',
   },
   {
+    code: 'eo',
+    emojiCode: ':star:',
```
Comment (andrewda) [inline src/auth/screens/language-settings.js]: I think this should probably be a flag, even if it's `:waving_white_flag:`.

Comment (housseindjirdeh) [reply src/auth/screens/language-settings.js]: Yeah @lex111 and i have discussed how we can possibly change this, but in the meantime `:waving_white_flag:` should do :)

**Comment by housseindjirdeh** [comment]
Thank you @kodisto 🙏  Only thing is the flag emoticon, everything else looks solid 🎉

**Comment by chinesedfan** [comment]
@housseindjirdeh This PR breaks ESLint. I will fix it in #478.

**Pull request merged**